library/common-library/templates/_cluster.tpl

```diff
@@ -9,8 +9,16 @@ Return the cluster
     {{- .Values.cluster -}}
 {{- else if $global.cluster -}}
     {{- $global.cluster -}}
-{{- else -}}
+{{- else if (include "common.cluster.failIfEmpty" . ) -}}
```

Comment:
@paologallinaharbur made me remove this from the licenseKey helper please yell at @kang-makes as well

Reply:
I will take a look at `nri-kubernetes` and metrics adapter for unused functions and I can/will remove them when I write/review the style guide.

Reply:
The overriding system was removed so this reminder is no longer needed.

Solving.